packages/demo-explorer/src/pages/CredentialVerifier.tsx

```diff
@@ -75,11 +75,16 @@ const CredentialVerifier = () => {
           credential: JSON.parse(text),
           fetchRemoteContexts: true,
         })
-        setVerificationResult(result)
+
         if (result?.verified) {
-          setVerificationResult((result) => {
-            if (!result) return result
-            return { ...result, verifiableCredential: JSON.parse(text) }
+          setVerificationResult({
+            ...result,
+            verifiableCredential: JSON.parse(text),
+          })
+        } else {
+          setVerificationResult({
+            verified: false,
+            error: { message: result?.error.errors[0].message },
```

Comment:
Should handle exception data. How about `JSON.stringify(result?.error, null, 2)`